### PR TITLE
near/tooling: Fix up deployment tool

### PR DESCRIPTION
**Pull request by jumpsiegel**

### CODE REVIEW

File: clients/js/main.ts [hunk 1]
```diff
@@ -496,6 +505,91 @@ yargs(hideBin(process.argv))
...
+            describe: "attach some near",
+            type: "string",
+            required: false,
+          })
+          .option("target", {
+            describe: "near account to upgrade",
+            type: "string",
+            required: false,
+          })
+          //            .option("rpc", {
```
Comment (panoel) [inline clients/js/main.ts:545]: Remove commented code?

File: clients/js/near.ts [hunk 2]
```diff
@@ -1,10 +1,112 @@
...
 const BN = require("bn.js");
 const nearAPI = require("near-api-js");
 
+function default_near_args(argv) {
+  let network = argv["n"].toUpperCase();
+  let contracts = CONTRACTS[network]["near"];
+  let n = NETWORKS[network]["near"];
+
+  if (!("rpc" in argv)) {
+    argv["rpc"] = n["rpc"];
```
Comment (panoel) [inline clients/js/near.ts:16]: Since rpc is commented out in main.ts, isn't this always the case?

File: clients/js/near.ts [hunk 3]
```diff
@@ -1,10 +1,112 @@
...
+  }
+
+  if (!("key" in argv)) {
+    if (n["key"]) {
+      argv["key"] = n["key"];
+    }
+  }
+
+  if (!("key" in argv)) {
+    if ("mnemonic" in argv) {
```
Comment (panoel) [inline clients/js/near.ts:37]: Can this be combined with the previous block (lines 30-34)?

**Review by panoel — APPROVED** [review]

File: clients/js/near.ts (hunk 2 above)
Comment (jumpsiegel) [reply clients/js/near.ts:16]: the rpc can come from a higher level argument setting..

**Review by panoel — APPROVED** [review]

**Review by kcsongor — APPROVED** [review]

**Pull request merged**